client/src/components/App/AppToolbar/TopToolbar/TopToolbar.tsx

```diff
@@ -13,9 +15,17 @@ const TopToolbar: React.FC = (): JSX.Element => {
                 <img alt='logo' className={classes.logo} src='./assets/img/logo.png'></img>
                 <Typography variant='h4'><div className={classes.systemName}>שם המערכת</div></Typography>
             </div>
-            <Typography>שלום, שם שלך</Typography>
+            <Typography>שלום, {props.userName}</Typography>
         </Toolbar>
     )
 }
 
-export default TopToolbar;
+const mapStateToProps = (state: StoreStateType) => ({
+    userName: state.user.name
+});
+
+interface Props {
+    userName: string
+}
+
+export default connect(mapStateToProps)(TopToolbar);
```

Comment:
You can use the useSelector hook from redux and remove the connect and the props mapping